.circleci/config.yml

```diff
@@ -1,36 +1,75 @@
 version: 2
 
+unit_tests: &unit_tests
+  steps:
+    - checkout
+    - run:
+        name: Create Checksum
+        command: sh .circleci/checksum.sh /tmp/checksums.txt
+    - restore_cache:
+        keys:
+          - npm-cache-{{ checksum "/tmp/checksums.txt" }}
+    - run:
+        name: Install Dependencies
+        command: yarn install
+    - save_cache:
+        key: npm-cache-{{ checksum "/tmp/checksums.txt" }}
+        paths:
+          - ./node_modules
+          - ./yarn.lock
+          - ./packages/*/node_modules
+          - ./packages/*/yarn.lock
+    - run:
+        name: Compile code
+        command: yarn compile
+    - run:
+        name: Unit tests
+        command: yarn test
+    - run:
+        name: report coverage
+        command: yarn codecov
+
 jobs:
-  build:
+  lint:
     docker:
-      - image: circleci/node:12-browsers
+      - image: node
     steps:
       - checkout
       - run:
-          name: Create Checksum
-          command: sh .circleci/checksum.sh /tmp/checksums.txt
-      - restore_cache:
-          keys:
-            - npm-cache-{{ checksum "/tmp/checksums.txt" }}
-      - run:
-          name: Install Dependencies
+          name: Install modules and dependencies.
           command: yarn install
-      - save_cache:
-          key: npm-cache-{{ checksum "/tmp/checksums.txt" }}
-          paths:
-            - ./node_modules
-            - ./yarn.lock
-            - ./packages/*/node_modules
-            - ./packages/*/yarn.lock
       - run:
-          name: Lint
+          name: Check code style and linting
           command: yarn run check
-      - run:
-          name: Compile
-          command: yarn compile
-      - run:
-          name: Test
-          command: yarn test
-      - run:
-          name: report-coverage
-          command: yarn codecov
+  node8:
+    docker:
+      - image: node:8
```

Comment:
For the node docker images that don't have browsers installed, we will need a separate test command (or environment variable) to disable the browser tests, since they run on `yarn test`, see https://github.com/open-telemetry/opentelemetry-js/blob/669b088bda51efa42bc07d4731fbb0eea4b5751e/packages/opentelemetry-core/package.json#L12

Maybe we need `yarn test:node` and `yarn run test:browser` and segment the tests in this file by that?

Reply:
I like the idea, made the appropriate changes in the config file. ptal

Reply:
Sweet! Now the Node and browser tests can also run in parallel!